drupal8-apache/7.0/usr/local/share/drupal/drupal8_functions.sh

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env bash
+
+#####
+# Perform the Drupal 8 build.
+#####
+do_drupal_build() {
+  do_drupal_create_directories
+  do_drupal_permissions
+  do_drupal_legacy_install_script
+}
+
+do_drupal_development_build() {
+  do_drupal_build
+  do_drupal_legacy_development_install_script
+}
+
+#####
+# Perform tasks on container start
+#####
+do_drupal_start() {
+  do_drupal_permissions
+  do_drupal_legacy_install_finalise_script
+}
+
+do_drupal_development_start() {
+  do_drupal_start
```

Comment:
`check_development_start` that triggers `do_development_start` when in dev mode is triggered from `do_start` originally (see https://github.com/continuouspipe/dockerfiles/blob/0e9a3f71562952f50774772c4faa2c53be20bb1e/ubuntu/16.04/usr/local/share/container/baseimage-10.sh#L15 )

That means the do_drupal_start will get triggered anyway, so no need to repeat.

If `do_drupal_build` where here instead, we would re-run the dev install steps on container start which is what we want for local development when /app is a mountpoint.